analyzer/codechecker_analyzer/cmd/analyze.py

```diff
@@ -347,7 +347,6 @@ def add_arguments_to_parser(parser):
                                dest='analyzers',
                                metavar='ANALYZER',
                                required=False,
-                               choices=analyzer_types.supported_analyzers,
                                default=argparse.SUPPRESS,
                                help="Run analysis only with the analyzers "
```

Comment:
The help message could be extended by some info about the version number. Don't forget to apply the changes in `docs/analyzer/user_guide.md` too.